hardinfo/util.c

```diff
@@ -979,6 +994,9 @@ gint tree_view_get_visible_height(GtkTreeView * tv)
 
 void tree_view_save_image(gchar * filename)
```

Comment:
This whole function can be removed, it's not used for anything. (It's a leftover from a feature I've never finished.)